projects/plugins/jetpack/composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "11ef58a0d1c39e8db4dd6a777bb40ccf",
+    "content-hash": "af17c209dbbca5ff6c8d3d2520eed30b",
```

Comment:
I don't know if this should be ported back, since we're still using the dev version in the `master` branch?

Reply:
```suggestion
    "content-hash": "11ef58a0d1c39e8db4dd6a777bb40ccf",
```